content/en/docs/File Format/implementationstatus.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Implementation status"
+linkTitle: "Implementation status"
+weight: 8
+---
+### Physical types
```

Comment:
it might help to provide guidance on how this should be filled in.  e.g. R for Read Support, W for Write Support and a date or version that it that support started on?

Reply:
+1 on giving guidance. I would support either date or version, though "date of the first release supporting it" sounds better to me (also because at some point I would like to propose that we define calendar-based compatibility profiles).